app/src/i18n/server.ts

```diff
@@ -1,32 +1,27 @@
-import { createInstance } from "i18next";
-import resourcesToBackend from "i18next-resources-to-backend";
-import { getOptions } from "@/i18n/settings";
+import i18next from "i18next";
+import { getOptions } from "./settings";
+import enEmails from "./locales/en/emails.json";
+import deEmails from "./locales/de/emails.json";
+import esEmails from "./locales/es/emails.json";
+import frEmails from "./locales/fr/emails.json";
+import ptEmails from "./locales/pt/emails.json";
 
-const initI18next = async (lng: string, ns: string) => {
-  const i18nInstance = createInstance();
-  await i18nInstance
-    .use(
-      resourcesToBackend(
-        (language: string, namespace: string) =>
-          import(`./locales/${language}/${namespace}.json`),
-      ),
-    )
-    .init(getOptions(lng, ns));
-  return i18nInstance;
+// Create a static resource map
+const resources = {
+  en: { emails: enEmails },
+  de: { emails: deEmails },
+  es: { emails: esEmails },
+  fr: { emails: frEmails },
+  pt: { emails: ptEmails },
 };
 
-export async function translationFunc(
-  lng: string,
-  ns: string,
-  options: { keyPrefix?: string } = {},
-) {
-  const i18nextInstance = await initI18next(lng, ns);
-  return {
-    t: i18nextInstance.getFixedT(
-      lng,
-      Array.isArray(ns) ? ns[0] : ns,
-      options.keyPrefix,
-    ),
-    i18n: i18nextInstance,
-  };
-}
+// Initialize i18next for server-side use
+i18next.init({
+  ...getOptions(),
+  resources,
+  lng: "en", // default language
+  fallbackLng: "en",
+  preload: ["en", "de", "es", "fr", "pt"], // preload all supported languages
```

Comment:
This needs to be imported from `src/i18n/settings.ts` so we only have to adjust one source of truth when we add languages. In general, make sure you're loading those settings in the same way in the i18next constructor as the client version does so we can prevent bugs caused by inconsistencies in the configs.